modules/ml/src/main/java/org/apache/ignite/ml/nn/architecture/MLPArchitecture.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.ml.nn.architecture;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.ignite.ml.math.functions.IgniteDifferentiableDoubleToDoubleFunction;
+
+/**
+ * Class containing information about architecture of MLP.
+ */
+public class MLPArchitecture {
+    /**
+     * List of layers architectures.
+     */
+    private final List<LayerArchitecture> layers;
+
+    /**
+     * Construct an MLP architecture.
+     *
+     * @param inputSize Size of input.
```

Comment:
What's input there?

Reply:
Count of inputs to MLP i.e. neurons in 0th layer.